test/system/surplus-for-auction.test.js

```diff
@@ -0,0 +1,182 @@
+const { expect } = require("chai")
+const {
+  to1e18,
+  impersonateAccount,
+  resetFork,
+  to1ePrecision,
+  ZERO_ADDRESS,
+} = require("../helpers/contract-test-helpers")
+const Auction = require("../../artifacts/contracts/Auction.sol/Auction.json")
+
+const describeFn =
+  process.env.NODE_ENV === "system-test" ? describe : describe.skip
+
+// This system test checks a scenario where we create a cov pool auction for the
+// deposits being under liquidation. There are two bidders come into the picture,
+// where the first one partially takes an offer and the second one buys the same
+// deposit outside the coverage pool. Next step is notifying the risk manager
+// about liquidated deposit so it can early close the auction.
+// All the tests below are executed on Hardhat Network with mainnet forking enabled.
+// At the start, the fork is being reset to the specific starting block which
+// determines the initial test state. This test uses the real tBTC token contract
+// and two deposits:
+// https://allthekeeps.com/deposit/0x8495732aecd7f132eaab61f64858ccc73475973f 5 TBTC
+// https://allthekeeps.com/deposit/0xfc9c50fd44879bd7085edd311bc8e2b7d3e41595 1 TBTC
+// which are ready to be liquidated at the starting block. All the bidders are also
+// real accounts with actual TBTC balance. At the end of the scenario, the risk
+// manager should early close the cov pool auction and keep the surplus of TBTC.
```

Comment:
I would consider adding a sentence about the second deposit being bought by the cov pool using the surplus TBTC.

Reply:
Sure 7eeb4c7